APMrover2/Log.cpp

```diff
@@ -284,11 +311,11 @@ void Rover::Log_Write_Attitude()
     DataFlash.Log_Write_Attitude(ahrs, targets);
 
 #if AP_AHRS_NAVEKF_AVAILABLE
- #if defined(OPTFLOW) and (OPTFLOW == ENABLED)
+#if defined(OPTFLOW) and (OPTFLOW == ENABLED)
```

Comment:
for nested # I always prefer a space or a tab indention. I don't care which one but having them all left justified is hard to read. Also, that "and" stuff is horrible and it should be &&